src/rf/js/src/home/controllers.js

```diff
@@ -33,16 +25,65 @@ var UserController = {
         var props = {
             handleLogin: function(e) {
                 e.preventDefault();
-                _logged_in = true;
-                router.navigate('/', {trigger: true});
+                router.go('/');
             }
         };
 
         var el = $('#container').get(0);
-        React.render(<views.LoginView {...props} />, el);
+        React.render(<Login {...props} />, el);
+    },
+
+    logout: function() {
+        router.go('/login');
     }
 };
 
+// Copied from prototype.
+function bindEvents() {
```

Comment:
I could not think of a good way to combine this easily with the React component. Maybe we do need to use Backbone views after all.